PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.h

```diff
@@ -66,12 +82,20 @@ class AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA : public AliAnalysisTaskEmcalJ
     void                        DoTrackLoop()                                     ;
     void                        DoClusterLoop()                                   ;
     void                        DoCellLoop()                                      ;
+    Bool_t                      IsLEDEvent() const                                ;
     Bool_t                      fUseRecalcPatches;          ///< Switch between offline (FEE) and recalc (L1) patches
     Bool_t                      SelectSingleShowerPatch(const AliEMCALTriggerPatchInfo *patch) const;
     Bool_t                      SelectJetPatch(const AliEMCALTriggerPatchInfo  *patch) const;
     THistManager                fHistManager                                      ;///< Histogram manager
+    Double_t                    GetZ(const Double_t trkPx, const Double_t trkPy, const Double_t trkPz, const Double_t jetPx, const Double_t jetPy, const Double_t jetPz) const;
+    Double_t                    GetSmearedTrackPt(AliVTrack *track)                ; ///< smeared Pt resolution with gaussian
 
 private:
+    
+    AliEMCALGeometry              *fGeom                     ;//!          EMCal goemetry utility
```

Comment:
This shadows the base class. And you should update the streaming comment (either `///<` or `//!<!`